### PR TITLE
feat/EntityRestaurantMenu

### LINKED ISSUE #11: Creazione Entity MenuRistorante

### PR DESCRIPTION
**Pull request by AleCam04**

### CODE REVIEW

File: src/main/java/com/develhope/Java27_progetto3_team2/entity/RestaurantMenu.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Entity
```
Comment (dudinhab) [inline src/main/java/com/develhope/Java27_progetto3_team2/entity/RestaurantMenu.java]: Entity name is missing

File: src/main/java/com/develhope/Java27_progetto3_team2/model/RestaurantMenuDTO.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.develhope.Java27_progetto3_team2.model;
+
```
Comment (dudinhab) [inline src/main/java/com/develhope/Java27_progetto3_team2/model/RestaurantMenuDTO.java:2]: Methods appropriate Getters and Setters are missing

File: src/main/java/com/develhope/Java27_progetto3_team2/entity/RestaurantMenu.java (hunk 1 above)
Comment (DonatoAletta) [reply src/main/java/com/develhope/Java27_progetto3_team2/entity/RestaurantMenu.java]: Also missing @AllArgsConstructor, @NoArgsConstructor

File: src/main/java/com/develhope/Java27_progetto3_team2/model/RestaurantMenuDTO.java (hunk 2 above)
Comment (DonatoAletta) [reply src/main/java/com/develhope/Java27_progetto3_team2/model/RestaurantMenuDTO.java:2]: Also missing @AllArgsConstructor, @NoArgsConstructor

File: src/main/java/com/develhope/Java27_progetto3_team2/entity/RestaurantMenu.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+public class RestaurantMenu {
+   @Id
```
Comment (DonatoAletta) [inline src/main/java/com/develhope/Java27_progetto3_team2/entity/RestaurantMenu.java:15]: Missing @GeneratedValue(strategy = GenerationType.IDENTITY)

File: src/main/java/com/develhope/Java27_progetto3_team2/entity/RestaurantMenu.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+   private int id;
+   @Setter
+   @Column(name = "menu")
+   private String menu;
```
Comment (dudinhab) [inline src/main/java/com/develhope/Java27_progetto3_team2/entity/RestaurantMenu.java]: private List<menuItem> menuItem

**Review by DonatoAletta — APPROVED** [review]

**Review by Fabio013 — APPROVED** [review]
Pr ok

**Pull request merged**